sentry-uitest/sentry-uitest-android-benchmark/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="io.sentry.uitest.android.benchmark">
+
+    <!--
+      Important: disable debugging for accurate performance results
+
+      In a com.android.library project, this flag must be disabled from this
+      manifest, as it is not possible to override this flag from Gradle.
+    -->
+    <application
+      android:theme="@style/Theme.AppCompat"
+      android:label="Sentry Benchmark"
+      android:icon="@android:mipmap/sym_def_app_icon"
+      android:debuggable="false"
```

Comment:
Isn't `android:debuggable` already set in the Gradle file anyway?

Reply:
yes, initially the module was an android library, not an application, so i had to put debuggable flag in the manifest (gradle doesn't accept it for libraries).
I forgot to remove it